.clang-format

```diff
@@ -1,9 +1,5 @@
 ---
-# Custom options in the special build of clang-format (these are not standard options)
-IndentNestedBlocks: false
-AllowNewlineBeforeBlockParameter: false
```

Comment:
these two options come from this patch https://reviews.llvm.org/D17700

Reply:
Which never landed, and were functionally replaced by the added ObjC block option